lrlex/src/lib/ctbuilder.rs

```diff
@@ -794,7 +815,15 @@ where
     ///
     /// Setting this flag will override the same flag within a `%grmtools` section.
     pub fn dot_matches_new_line(mut self, flag: bool) -> Self {
-        self.force_lex_flags.dot_matches_new_line = Some(flag);
+        let key = "dot_matches_new_line".to_string();
+        self.header.set_merge_behavior(&key, MergeBehavior::Ours);
```

Comment:
We could add a function here `set_default_merge_behavior`, and avoid adding this line in all of the setters here.

Reply:
I implemented this in [b399a72](https://github.com/softdevteam/grmtools/pull/556/commits/b399a721e8b78619a9fff81f7fae34172a38aed3) but happy to back it out if you would prefer the minimalist API or would implementing it as a separate PR.

Reply:
Works for me!